NEWS.md

```diff
@@ -1,6 +1,41 @@
 # NEWS
 
-## 0.24
+## 0.24.1
+
+### Feature Management
+
+This release supports the upcoming Spec Version 0.0.4 that enables subgraph features to be declared in the manifest and
+validated during subgraph deployment
+[#2682](https://github.com/graphprotocol/graph-node/pull/2682)
+[#2746](https://github.com/graphprotocol/graph-node/pull/2746).
+
+> Subgraphs using previous versions are still supported and won't be affected by this change.
+
+### Api Version 0.0.5
+
+- Added better error message for null pointers in the runtime [#2780](https://github.com/graphprotocol/graph-node/pull/2780).
+
+### Environment Variables
+
+- Added `GRAPH_STORE_CONNECTION_MIN_IDLE` for setting the minimum number of allowed database connections.
+- Added `GRAPH_STORE_CONNECTION_IDLE_TIMEOUT` for setting the timeout of when a database connection is considered idle.
```

Comment:
We should not document/mention these two - if these settings have no or little effect on the hosted service, we'll remove them again